src/Employer/Employer.Web/wwwroot/javascripts/application.js

```diff
@@ -164,4 +157,59 @@ $(window).load(function() {
     if ($('.error-summary').length) {
       $('.error-summary').focus();
     } 
+});
+
+function handleAnchorClicks() {
+    if (document.documentElement.scrollIntoView) {
+
+        var $menu = $("#floating-menu-holder .account-information");
+
+        $(".review-anchor-link").on("click", function (e) {
+            
+            var hash = $(this)[0].hash;
+            var $element = $(hash);
```

Comment:
Misunderstood, so `$(this)[0]` is a window.location element?

The reason I suggested above was more to remove the `this` keyword and acting in context. Mainly because I hate scriptJava 👿